lambdas/check_new_areas/function.py

```diff
@@ -11,47 +11,59 @@
 from shapely.wkb import dumps
 from shapely.geometry import shape, Polygon
 
+from geotrellis_summary_update.secrets import get_token
+from geotrellis_summary_update.exceptions import EmptyResponseException
+
 # environment should be set via environment variable. This can be done when deploying the lambda function.
 if "ENV" in os.environ:
     ENV = os.environ["ENV"]
 else:
     ENV = "dev"
 
 
-class EmptyResponseException(Exception):
-    pass
-
-
-def secret_suffix() -> str:
-    if ENV == "production":
-        suffix: str = "prod"
-    else:
-        suffix = "staging"
-    return suffix
-
-
 S3_CLIENT = boto3.client("s3")
-SM_CLIENT = boto3.client("secretsmanager")
-TOKEN: str = SM_CLIENT.get_secret_value(SecretId=f"gfw-api/{secret_suffix()}-token")
+TOKEN: str = get_token(ENV)
+PENDING_AOI_NAME = "pending_user_areas"
+PENDING_AOI_ANALYSES = {
+    "dev": {
+        "annualupdate_minimal": {
+            "change": "206938be-12d9-47b7-9865-44244bfb64d6",
+            "summary": "0eead72d-1ad7-4c0f-93c4-793a07cd2e3d",
+        },
+        "gladalerts": {
+            "daily_alerts": "722d90b2-e989-48ca-ba27-f7a8e236ea44",
+            "weekly_alerts": "153a2ba7-cff6-4b06-bd76-279271c4ddea",
+            "summary": "28e44bd2-cd13-4587-9259-1ba235a82d28",
+        },
+    }
+}
```

Comment:
I don't think, this is the right location for this information. this lambda function does not deal with API datasets at all. ID's are also env depended and might change. 
I assume, IDs are needed for functions further downstream. I suggest to create a lookup table which holds the latest IDs which we can query inside the function that needs it.

Reply:
Yeah the IDs are just used downstream, but also provides the config for which analyses to run and the different results. But yeah, I think it should be defined in externally.

Reply:
Actually, I remembered why I put it like this: the lambdas that submit the EMR job and then update the datasets are parted of their own core step function. So, this lambda essentially just invokes that step function, and that's why it just passes all the information, including the downstream datasets. (in this case, it invokes it via a nested step function in a larger step function, see step_functions/summarize_new_areas.json)

I designed it that way so the core flow can be called independently, either as other cron jobs or one offs. So I think it makes more sense to still pass all this info in to that core step function. If we have some other lookup table each lambda looks at, then calling the core step function requires updating that table every time, which doesn't make sense for one-off cases. But I'll remove it as a constant and put it in a YAML file.

Reply:
Sounds good for now. Please see #9 for a better solution, once we are ready.